hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -1587,6 +1587,15 @@ private void initializeRatisDirs(OzoneConfiguration conf) throws IOException {
     omRatisSnapshotDir = OmUtils.createOMDir(
         OzoneManagerRatisUtils.getOMRatisSnapshotDirectory(conf));
 
+    File omRatisDir = new File(omRatisDirectory);
```

Comment:
I'm wondering if the same mount point implies a hard link can be created. What if we try to create a hard link upon OM start (and then delete it)? That would be more direct proof that it works.